package.json

```
@@ -1,10 +1,10 @@
 {
   "name": "youtube-audio-server",
-  "version": "2.4.1",
+  "version": "2.6.0",
```

The prev. version was `2.4.x` and we're adding new features on this one.
We should probably just bump the minor in 1.
```suggestion
  "version": "2.5.0",
```